app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
@raluka could you describe in simple words what we want to achieve here? Because I think it makes perfect sense that the :manage option does not work... But please try to explain what we want to show and when, so that we can debug this!